README.md

```diff
@@ -47,7 +47,8 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 ## Index
 
 - [Android Development](#android-development)
-- [Backend](#backend)  
+- [Backend](#backend)
+  * [Shell Commands](#shell commands)
```

Comment:
Hey, Please replace this with `[Shell Commands](#shell-commands)` so, that it can redirect to its required location

Reply:
please review the changes i made